python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -342,28 +342,27 @@ def configure(self, masterappconfig):
         dir_name = self.directory
         return (None, None)
 
-
```

Comment:
PEP8 asks for two lines between global functions, but 1 between functions in classes.